### ISSUE TITLE
Feature suggestion: Introduce environment variable TABTAB_AUTO_PATH

### ISSUE THREAD

**Issue by yeonghoey** [mklabs/node-tabtab#39]
## TL;DR
How about providing `TABTAB_AUTO_PATH` option as an environment variable?
When it's defined, `--auto` installs/uninstalls its generated content into `$TABTAB_AUTO_PATH`.

## Long Story
I usually use `zsh` but also use `bash` in times and manage them under a git repo.
They only contain common settings and I put all the other local system dependent settings into `~/.localrc`.

As a result, my `.bashrc` and `.zshrc` both have following lines:
```sh
if [[ -f "$HOME/.localrc" ]]; then
  source "$HOME/.localrc"
fi
```
(It seems, as a matter of fact, that people who maintain their own `dotfiles` repos use patterns like this.)

Accordingly, I hope that every time I install some node packages which use `tabtab`, I want `tabtab` generated contents to be placed in `.localrc`.

**Comment by mklabs** [comment]
That's definitely something we can do. I like the idea.

Would you like to give it a try ? I would be more than happy to merge a PR implementing this. I can guide you through. I unfortunately don't have the time atm to work on this.

I'll eventually implement something similar, thanks for the feedback 👍

**Comment by yeonghoey** [comment]
It's a good thing you like the idea. But, currently I don't have any practical node.js experience, and even though I think this thing is an excellent opportunity for getting some, I think I cannot afford to work on it by now. I'll update here when I get some free time to work on this.

**Comment by mklabs** [comment]
No worries. I'll do my best to try remedy the issue
